enhancements/api-review/custom-tags-aws.md

```diff
@@ -0,0 +1,143 @@
+---
+title: apply-user-defined-tags-to-all-aws-resources-created-by-openshift
+authors:
+  - "@gregsheremeta"
+reviewers:
+  - @decarr
+  - @bparees
+approvers:
+  - @decarr 
+creation-date: 2021-03-24
+last-updated: 2021-04-09
+status: implementable
+---
+
+# Apply user defined tags to all AWS resources created by OpenShift
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
```

Comment:
@gregsheremeta Is this still accurate or have portions of this already been implemented? If so which? can you update this to detail that.